### ISSUE TITLE
camelcase shouldn't warn when reading properties

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#672]
Similar to #656, if you're reading a property value that has underscores in it, it shouldn't be your fault. A lot of web services do this, as well as libraries. So this should be okay:

``` js
var foo = bar.baz_boom;
```

This should still warn:

``` js
bar.baz_boom = "foo";
```
